samples/protected-setting-sample/mainTemplate.json

```diff
@@ -0,0 +1,249 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "extensionResourceName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the extension."
+            }
+        },
+        "clusterResourceName": {
+            "type": "String",
+            "metadata": {
+                "description": "The name of the Managed Cluster resource."
+            }
+        },
+        "createNewCluster": {
+            "type": "Bool",
+            "defaultValue": true,
+            "metadata": {
+                "description": "When set to 'true', creates new AKS cluster. Otherwise, an existing cluster is used."
+            }
+        },
+        "location": {
+            "type": "String",
+            "metadata": {
+                "description": "The location of AKS resource."
+            }
+        },
+        "app-title": {
+            "type": "String"
+        },
+        "app-value1": {
+            "type": "String"
+        },
+        "app-value2": {
+            "type": "String"
+        },
+        "password-element": {
+            "type": "securestring"
```

Comment:
Can you also highlight this part of the ARM template in README file? Add a new image just for the password element part. Users need to understand that they have to use type securestring.